src/main/scala/org/apache/spark/sql/delta/sources/DeltaSource.scala

```diff
@@ -371,4 +376,19 @@ case class DeltaSource(
       case other => throw new UnsupportedOperationException(s"Unknown ReadLimit: $other")
     }
   }
+
+  /** Extracts whether users provided the option to time travel a relation. */
+  private def getStartingVersion: Option[(Long, String)] = {
+    val tsOpt = options.startingTimestamp
+    val versionOpt = options.startingVersion
+
+    /** DeltaOption validates input and ensures that only one is provided. */
+    if (tsOpt.isDefined || versionOpt.isDefined) {
+      Some(DeltaTableUtils.resolveTimeTravelVersion(
+        spark.sessionState.conf, deltaLog,
+        DeltaTimeTravelSpec(tsOpt.map(Literal(_)), versionOpt, Some("streamSource"))))
```

Comment:
my bad: sorry, changed my mind here, can we change `streamSource` to `deltaSource` please